README.md

```diff
@@ -207,7 +207,12 @@ To install Fabric, you can use the latest release binaries or install it from th
 
 #### Windows
 
-`https://github.com/danielmiessler/fabric/releases/latest/download/fabric-windows-amd64.exe`
+```
```

Comment:
Change this to specify "powershell" as the language for the fenced code, please.